babel.config.js

```diff
@@ -123,7 +123,10 @@ module.exports = function(api) {
         ],
         plugins: [
           includeRuntime
-            ? ["@babel/transform-runtime", { version: "7.4.4" }]
+            ? [
+                "@babel/transform-runtime",
+                { version: require("@babel/runtime/package").version },
```

Comment:
Good idea

Reply:
Actually this option is _not_ effective until #10668 is merged, because we are always building against the `lib` version of `@babel/*`, which has inlined all the helper imports.